src/common/types/types.cpp

```diff
@@ -405,6 +406,7 @@ std::string LogicalTypeUtils::dataTypeToString(const LogicalType& dataType) {
     case LogicalTypeID::INTERVAL:
     case LogicalTypeID::STRING:
     case LogicalTypeID::SERIAL:
+    case LogicalTypeID::BLOB:
```

Comment:
If BLOB and STRING use the same code path, let's make sure we always put BLOB together with STRING. Change across this file.